tests/test_anon.py

```diff
@@ -0,0 +1,53 @@
+from pydicom import dcmread
+from pydicom.data import get_testdata_file
+from dicomanonymizer import dicomfields
+from dicomanonymizer import anonymize_dataset
+
+
+def test_pydicom_data():
+    sample_data = get_testdata_file("CT_small.dcm")
```

Comment:
Apparently you can use the named argument `read` to get a DataSet from `get_testdata_file` instead of getting a file that you have to read with `dcmread`

Reply:
Done